aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -310,8 +429,59 @@ Observe which of the `OperationId` values vary within a request and between requ
 
 Create an <xref:Microsoft.Extensions.DependencyInjection.IServiceScope> with [IServiceScopeFactory.CreateScope](xref:Microsoft.Extensions.DependencyInjection.IServiceScopeFactory.CreateScope*) to resolve a scoped service within the app's scope. This approach is useful to access a scoped service at startup to run initialization tasks. The following example shows how to obtain a context for the `MyScopedService` in `Program.Main`:
 
+::: moniker range=">= aspnetcore-3.0"
+
 ```csharp
-public static void Main(string[] args)
+// using System;
+// using System.Threading.Tasks;
+// using Microsoft.Extensions.DependencyInjection;
+// using Microsoft.AspNetCore.Hosting;
+// using Microsoft.Extensions.Hosting;
```

Comment:
Are these commented for a reason? Is it a docfx thing?

Reply:
They're outside of the `Program` class, and I was trying to vertically shorten the code.

Do you think I should add the `Program` class designation line? ... or just take the comments off? ... or what would be best? 🤔 ..... The `Program` line and one brace (`}`) at the end I guess wouldn't add much. Rubber :duck: says *Yah! Do that.* :smile: